.github/workflows/test.yml

```
@@ -40,7 +40,6 @@ jobs:
     env:
       # The "default" npm is the one that ships with a given version of node.
       # For more: https://nodejs.org/en/download/releases/
-      IS_MRT_NODE: ${{ matrix.node == 16 && matrix.npm == 8 }}
       IS_DEFAULT_NPM: ${{ matrix.node == 16 && matrix.npm == 8 || matrix.node == 18 && matrix.npm == 9 }}
```

```suggestion
      IS_DEFAULT_NPM: ${{ (matrix.node == 16 && matrix.npm == 8) || (matrix.node == 18 && matrix.npm == 9) }}
```
Just noticed I think we might need this.